kong-redis-cluster-1.4.0-0.rockspec

```diff
@@ -1,8 +1,8 @@
 package = "kong-redis-cluster"
-version = "1.3.0-1"
```

Comment:
please remove these changes. These belong in the release commit, not with code changes.